gromacs/gromacs-2022.x/colvarproxy_gromacs.cpp

```diff
@@ -47,9 +47,14 @@ void colvarproxy_gromacs::init(t_inputrec *ir, int64_t step, const gmx_mtop_t &m
 
   angstrom_value = 0.1;
 
+  // From Gnu units
+  // $ units -ts 'k' 'kJ/mol/K/avogadro'
+  // 0.0083144621
+  boltzmann_ = 0.0083144621;
+
   // Get the thermostat temperature.
   // NOTE: Considers only the first temperature coupling group!
-  thermostat_temperature = ir->opts.ref_t[0];
+  set_target_temperature(ir->opts.ref_t[0]);
```

Comment:
Could we use  directly the attribute `target_temperature_`? To be consistent with botlzmann above?

Reply:
Yeah, initially I wanted to use a function also for the Boltzmann constant, which could have been done via `set_unit_system()`, but it's better to leave it be for now.

Let me try adding `set_physical_constants()`, which could be a function that gets called within `set_unit_system()` when/if these are changed.

Reply:
Actually, scratch that, sorry. I don't want to mess with the units just yet. I'm going to set the Boltzmann constant from GROMACS as you suggested in the other comment.

For this comment specifically, I would prefer to keep the temperature (which is a user choice) separate from the physical constants (which are fixed and under the control of the unit system but not the user's currently). Better if these two are not consistent just yet.

Reply:
okay, fine by me!